dbt_odis/models/silver/silver_population_menages_region.sql

```diff
@@ -0,0 +1,20 @@
+{{ config(
+    tags = ['silver', 'population'],
+    alias='silver_population_menages_region'
+    )
+}}
+
+with population_nb_menages_reg as (
+    select
+        'reg' || geo."CODREG" as codgeo,
+        2021 as year,
```

Comment:
Si la suggestion a été appliquée dans la couche bronze sur la time_perid, alors refactorer ici:
```suggestion
        "year",
```